TestTables/CMIP7_atmos2d.json

```diff
@@ -0,0 +1,61 @@
+{
+    "Header":{
+        "Conventions":"CF-1.11 CMIP-7.0.0.0",
+        "checksum":"707fcfc8d9429acb47db0a00685b6901",
+        "cmor_version":"3.10",
+        "generic_levels":"alevel alevhalf",
+        "int_missing_value":"-999",
+        "missing_value":"1e20",
+        "ok_max_mean_abs":"",
+        "ok_min_mean_abs":"",
+        "positive":"",
+        "product":"model-output",
+        "realm":"atmos",
+        "table_date":"2025-03-05",
+        "table_id":"atmos2d",
+        "type":"real",
```

Comment:
`real` is a Fortran data type, if we're cleaning up, should map to modern C/Python types, most often being `float`

Reply:
From what I can ascertain, CMOR will only accept `real` (`float32`) and `integer` (`int32`) types, with no ability to more granularly use the space-saving data types of `byte/int8`, `ubyte/uint8`, `short/int16`, and `ushort/uint16` vs the default `real/float32` type

Reply:
that would be nice, but could be quite a big job, in FORTRAN at least.  Something, perhaps, later on.

Reply:
@mauzey1 it would be great if you can chime in on this query comment - how hard-coded are the data types in CMOR, and do we have the ability of using uint8 and other netCDF4 data types currently in CMOR?

Reply:
Migrated this discussion into https://github.com/PCMDI/cmor/issues/781